src/lib/github/issue-presenter.ts

```diff
@@ -20,4 +27,13 @@ const createIssueWithComments = (data: RepoIssue[]): IssueWithComments[] => {
   });
 };
 
-export default createIssueWithComments;
+const formattedIssueComment = (data: RepoIssue[]): Issue[] => {
+    const arrComments = createIssueWithComments(data);
+    return arrComments.map( (obj: IssueWithComments) => {
+          return ({
+            title: obj.title,
+            body: obj.comments.join(" -[] ")
```

Comment:
The first item in a comments list of more than one comment is going to miss  `-[]` if we only use join.

Reply:
each comment in the list should be formatted this way.

`const formattedComment  = `- [ ] ${commentText} \n`